### DISCUSSION TITLE
Win11 failure: "py-OpenSSL: missing", but it isn't

### DISCUSSION THREAD

**Discussion by SMcCandlish** [pyload/pyload#4431] [Help] [answered]
I run pyLoad's setup.exe, click Setup Assistant, and all tests are passed except it always says "py-OpenSSL: missing". But py-OpenSSL is NOT missing, and was installed with pip, in the standard distribution of Python 2.14 for Windows, at its default location.  In particular, it is at "C:\Program Files\Python312\Lib\ssl.py" where it belongs. I even added that directory to the system path, and your setup.exe still cannot find it. I'm not sure how to resolve this.

**Comment by SMcCandlish** [comment]
PS: I also did an OpenSSL build with `gvsbuild build openssl`, and have that working and in the path as "C:\gtk-build\gtk\x64\release\bin\openssl.exe".

**Comment by GammaC0de** [comment]
Python 3.12 is not supported, use python 3.11
See https://github.com/pyload/pyload#quick-start for installation instructions.

**Reply by SMcCandlish** [reply, marked as answer]
Nah. Downgrading Python would break too much other stuff. Guess I'll just have to wait.
